proxy-node-gateway/src/dist/config.yml

```diff
@@ -35,3 +35,5 @@ translatorService:
 redisService:
   local: ${USE_REDIS_LOCAL:-false}
   url: ${REDIS_SERVER_URI}
+
+errorPageRedirectUrl: ${PROXY_NODE_ERROR_PAGE_URI:-}
```

Comment:
This will need to be set in the deployment chart for Staging. You should talk to Rafal.

Reply:
Yep, that's right. For now this will work but it's set up correctly in staging and can test locally we can remove some more code from PN